userfiles/civet_output/civet_output.rb

```diff
@@ -89,5 +89,16 @@ def overlays #:nodoc:
     @overlays             = thickness_txts_for_select + surfaces_txts_for_select
   end
 
+  def dsid #:nodoc:
+    return @dsid unless @dsid.nil?
```

Comment:
Cache the value in the meta data store, that way we never have to synchronize ever again to get that value.